### PR TITLE
Louy

### PR DESCRIPTION
**Pull request by louy755**
Please check this page and if need any adjustment let me know

### CODE REVIEW

File: calculator/html.html [hunk 1]
```diff
@@ -0,0 +1,48 @@
+<!DOCTYPE html>
```
Comment (ssharba) [inline calculator/html.html]: all html files must be in the root directory.

File: calculator/css.css [hunk 2]
```diff
@@ -0,0 +1,70 @@
+/* Basic Reset*/
```
Comment (ssharba) [inline calculator/css.css]: css files must be in css folder.

File: calculator/js.js [hunk 3]
```diff
@@ -0,0 +1,72 @@
+function run1(){
```
Comment (ssharba) [inline calculator/js.js]: js files must be inside js folder

**Comment by louy755** [comment]
I did the requirements .

**Pull request closed without merging**